tests/cpu/test_gdn_correctness.py

```
@@ -0,0 +1,115 @@
+    torch.cpu.synchronize() # 确保所有计算完成
+    end_time = time.perf_counter()
+
+    # --- 步骤 D: 统计结果 ---
+    total_time_ms = (end_time - start_time) * 1000
+    avg_time_ms = total_time_ms / repeats
+    
+    # 计算吞吐量 (Tokens/s)。这里假设每个 batch 处理 1 个 token
+    # 实际吞吐量取决于你的业务定义，这里以 Batch Size 作为每秒处理的 token 数参考
+    tokens_per_second = (B * repeats) / (end_time - start_time)
+
+    print(f"\n📈 Benchmark 结果汇总:")
```

Ruff (0.15.17)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)